src/components/Header/Header.tsx

```diff
@@ -18,7 +23,12 @@ export const Header: React.FC = () => {
           <div className="header-comp pull-right">
             <HeaderBtn to={'index.html'} buttonType={'icon-bell'} />
             <HeaderBtn to={'index.html'} buttonType={'icon-mail'} />
-            <UserProfile alt={'userpic'} to={'index.html'} />
+            <UserProfile
+              alt={'userpic'}
+              to={'index.html'}
+              name={currentUser.user?.name}
```

Comment:
I think would be better to have optional render for the user profile component. (render something else if user not signed in, login button for example)